package/retris/package

```diff
@@ -0,0 +1,21 @@
+# vim: set ft=sh:
+pkgname=retris
+pkgver=0.5.3-1
+pkgdesc="Implementation of rust tetris_core on the reMarkable using libremarkable"
+section=games
+maintainer="Linus K. <linus@cosmos-ink.net>"
+license=MIT
+
+origin=https://github.com/LinusCDE/retris.git
+revision=0.5.3-1
```

Comment:
That's a very good idea. I even changed the tag sometime later to the latest commit, because I accidentally committed my custom ipkg. So yeah. They are very changeable which shouldn't be allowed here, indeed.